server/api/swagger/swagger.yaml

```diff
@@ -2809,6 +2809,59 @@ paths:
         - supervisor
         - support
         - kyc
+  /admin/user:
+    x-swagger-router-controller: admin
+    put:
+      operationId: putUserInfo
+      description: Update a user's info
+      tags:
+        - Admin
+      parameters:
+        - name: user_id
+          in: query
+          required: true
+          type: integer
+        - name: data
+          in: body
+          required: true
+          schema:
+            type: object
+            properties:
+              full_name:
+                type: string
+              gender:
+                type: boolean
+              nationality:
+                type: string
+              phone_number:
+                type: string
+              dob:
+                type: ['string', 'null']
+                format: date-time
+              address:
+                type: object
+                properties:
+                  country:
+                    type: string
+                  address:
+                    type: string
+                  postal_code:
+                    type: string
+                  city:
+                    type: string
+      responses:
+        200:
+          description: Success
+          schema:
+            $ref: "#/definitions/UserResponse"
+        default:
+          description: Error
+          schema:
+            $ref: "#/definitions/MessageResponse"
+      security:
+        - Bearer: []
+      x-security-scopes:
+        - admin
```

Comment:
add supervisor here as well @brandonkimmmm

Reply:
@abeikverdi Made the update below